public/app/features/alerting/unified/types/amroutes.ts

```diff
@@ -2,7 +2,7 @@ import { MatcherFieldValue } from './silence-form';
 
 export interface FormAmRoute {
   id: string;
-  matchers: MatcherFieldValue[];
```

Comment:
Note: I'm not sure this can safely be removed. Grafana Managed alerts will use `object_matchers` but the non-Grafana managed I think will still return `matchers`